### PR TITLE
Feature/stud 114/settings section

### PR DESCRIPTION
**Pull request by yasfahm**
IMG_2DEE8EAB0C7B-1

### CODE REVIEW

**Comment by yasfahm** [comment]
Added save button!
IMG_44194AF4EA18-1

**Comment by stellakimnguyen** [comment]
Everything looks great!! Potentially fix the codacy issues (only two missing semicolons) and you should be to goo

File: src/components/SaveButton.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  // STYLED-COMPONENTS
+const CustomSaveButton = styled.TouchableOpacity `
+padding-top: 20
+alignItems: center;
+`;
+
+const CustomSaveIcon = styled.Image `
+tintColor: ${theme.COLOR_GREEN};
+width: 30; 
+height: 30;
```
Comment (stellakimnguyen) [inline src/components/SaveButton.js]: Not sure if it's github, but seems to be a missing indent for lines 24-25 and 29-31

File: src/components/ToggleButton.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        value: true // keep it true for now
+      };
+    }
+
+    render() {
+      return (
+        <ToggleContainer>
+            <Label>{this.props.labelName}</Label>
+            <CustomSwitch
+              style={{ transform: [{ scaleX: 0.6 }, { scaleY: 0.6 }] }}
```
Comment (stellakimnguyen) [inline src/components/ToggleButton.js]: You could add this directly into your CustomSwitch styled-component!

File: src/containers/TextInputContainer.js [hunk 3]
```diff
@@ -1,27 +1,59 @@
...
+  }
 }
 
 // PROP-TYPES
 TextInputContainer.propTypes = {
-  children: PropTypes.element.isRequired,
 };
 
+const Container = styled.View`
+    flexDirection: row;
```
Comment (stellakimnguyen) [inline src/containers/TextInputContainer.js]: (again not sure if it's github) but indent might seem off compared to the ones below?

**Review by stellakimnguyen — APPROVED** [review]

File: src/components/ToggleButton.js (hunk 2 above)
Comment (yasfahm) [reply src/components/ToggleButton.js]: added!

**Pull request merged**